.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# This file assigns responsibility for reviewing specific files
+# in this repo to specific people or teams.
+# See: https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/about-code-owners
+
+*	@joemcgill
```

Comment:
Fine for now, but wonder if this should be the alley admins group?

Reply:
Sames safe enough to me